### ISSUE TITLE
[Push Publishing] : Push Publishing Filters being created with wrong file name

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#23578]
Reproducible in latest `master`.

### Describe the bug
Push Publishing Filters must be created under the `assets/server/publishing-filters/`. However, there seems to be an error when such YML files are generated, and result in the following format:
```
/assets/server/publishing-filtersContentOnly.yml
/assets/server/publishing-filtersForcePush.yml
/assets/server/publishing-filtersIntelligent.yml
/assets/server/publishing-filtersShallowPush.yml
/assets/server/publishing-filtersWebContentOnly.yml
```
Notice how the `publishing-filters` folder name is being appended to the YML file name.

### To Reproduce
1. Start up dotCMS.
2. Check the folder where your assets are being generated, and check the `assets/server/` folder.
3. Notice how the `/publishing-filters/` folder is empty, and the presence of the incorrect YML files.

### Expected behavior
The Push Publishing Filter files must be created in the right location for both dotCMS and Postman/Integration Tests to work correctly.

**Comment by jcastro-dotcms** [comment]
The error is caused by the following changes: https://github.com/dotCMS/core/blame/master/dotCMS/src/main/java/com/dotcms/publishing/PublisherAPIImpl.java#L59

This is causing the trailing slash to be removed from the path, which is causing the folder to be appended to the YAML's file name.

**Comment by fabrizzio-dotCMS** [comment]
PR https://github.com/dotCMS/core/pull/23583

**Comment by bryanboza** [comment]
Fixed, tested on release 23.02 // Docker // FF
